snmp/datadog_checks/snmp/data/default_profiles/_hp.yaml

```diff
@@ -0,0 +1,8 @@
+extends:
+ - _base.yaml
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "hp"
```

Comment:
🥜 Could you configure your IDE to automatically add trailing newline at the end of a file.

It's not critical, but here are few reasons why + on how to configure on IDE: https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3